### PR TITLE
Removed swipe right to go back

### PR DESCRIPTION
**Pull request by hcarneiro**

### CODE REVIEW

File: interface.html [hunk 1]
```diff
@@ -21,5 +21,24 @@
       </label>
       </div>
     </div>
+    <div class="form-group clearfix">
+      <div class="col-sm-4 control-label">
+        <label>Do you want to be able to swipe right to go back a screen?</label>
+      </div>
```
Comment (tonytlwu) [inline interface.html]: All this still needs to be removed

**Review by tonytlwu — CHANGES_REQUESTED** [review]
All this still needs to be removed

File: interface.html (hunk 1 above)
Comment (hcarneiro) [reply interface.html]: @tonytlwu Approval needed again!

**Review by tonytlwu — APPROVED** [review]

**Pull request merged**